sdk/keyvault/arm-keyvault/CHANGELOG.md

```diff
@@ -1,44 +1,95 @@
   - Added Interface ManagedHsmsUpdateHeaders
+  - Added Interface ManagedHsmTrigger
+  - Added Interface MhsmGeoReplicatedRegion
   - Added Interface MhsmPrivateEndpointConnection
   - Added Interface MhsmPrivateLinkResource
+  - Added Interface MhsmRegionsListByResourceNextOptionalParams
+  - Added Interface MhsmRegionsListByResourceOptionalParams
+  - Added Interface MhsmRegionsListResult
   - Added Interface PrivateEndpointConnection
   - Added Interface PrivateLinkResource
+  - Added Interface ProxyResourceWithoutSystemData
   - Added Interface RotationPolicy
   - Added Interface Secret
   - Added Interface SecretAttributes
   - Added Interface Trigger
+  - Added Type Alias ActivationStatus
+  - Added Type Alias GeoReplicationRegionProvisioningState
   - Added Type Alias KeyRotationPolicyActionType
+  - Added Type Alias ManagedHsmKeysCreateIfNotExistResponse
+  - Added Type Alias ManagedHsmKeysGetResponse
+  - Added Type Alias ManagedHsmKeysGetVersionResponse
+  - Added Type Alias ManagedHsmKeysListNextResponse
+  - Added Type Alias ManagedHsmKeysListResponse
+  - Added Type Alias ManagedHsmKeysListVersionsNextResponse
+  - Added Type Alias ManagedHsmKeysListVersionsResponse
+  - Added Type Alias ManagedHsmsCheckMhsmNameAvailabilityResponse
+  - Added Type Alias ManagedHsmsPurgeDeletedResponse
+  - Added Type Alias MhsmRegionsListByResourceNextResponse
+  - Added Type Alias MhsmRegionsListByResourceResponse
   - Interface KeyProperties has a new optional parameter releasePolicy
   - Interface KeyProperties has a new optional parameter rotationPolicy
+  - Interface ManagedHsmProperties has a new optional parameter regions
+  - Interface ManagedHsmProperties has a new optional parameter securityDomainProperties
   - Interface MhsmPrivateEndpointConnectionItem has a new optional parameter etag
   - Interface MhsmPrivateEndpointConnectionItem has a new optional parameter id
+  - Added Enum KnownActivationStatus
+  - Added Enum KnownGeoReplicationRegionProvisioningState
   - Enum KnownJsonWebKeyOperation has a new value Release
   - Enum KnownKeyPermissions has a new value Getrotationpolicy
   - Enum KnownKeyPermissions has a new value Release
   - Enum KnownKeyPermissions has a new value Rotate
   - Enum KnownKeyPermissions has a new value Setrotationpolicy
+  - Added function getContinuationToken
+  - Interface SecretsListNextOptionalParams no longer has parameter top
+  - Interface VaultsListByResourceGroupNextOptionalParams no longer has parameter top
+  - Interface VaultsListBySubscriptionNextOptionalParams no longer has parameter top
+  - Interface VaultsListNextOptionalParams no longer has parameter top
+  - Interface ManagedHsmsListByResourceGroupNextOptionalParams no longer has parameter top
+  - Interface ManagedHsmsListBySubscriptionNextOptionalParams no longer has parameter top
+
+**Breaking Changes**
+
+  - Interface MhsmPrivateEndpointConnectionsDeleteHeaders no longer has parameter retryAfter
```

Comment:
Need to check the swagger source

Reply:
https://github.com/Azure/azure-rest-api-specs/pull/20839

Reply:
I don't think this is introduced here, could you try to find the swagger position?